robottelo/cli/hostcollection.py

```diff
@@ -35,25 +37,42 @@ class HostCollection(Base):
     command_base = 'host-collection'
 
     @classmethod
+    def transform_ids(cls, options=None):
+        """Workaround host unification feature not being completed to use host
+        id instead content host uuid
```

Comment:
And of course, current code is not unique and similar actions performed for other modules, for example:
`def delete(cls, options=None):` in `cli/contenthost.py`